modules/apps/adaptive-media/adaptive-media-web/src/main/java/com/liferay/adaptive/media/web/internal/servlet/AMServlet.java

```diff
@@ -100,6 +102,19 @@ protected void doGet(
 
 			String fileName = fileNameOptional.orElse(null);
 
+			String uri = String.valueOf(adaptiveMedia.getURI());
```

Comment:
I think is better to declare

URI uri = adaptiveMedia.getURI();
and `long fileEntryId = _getFileEntryId(uri.toString());` or `long fileEntryId = _getFileEntryId(String.valueOf(uri)` or inlining it
`long fileEntryId = _getFileEntryId(String.valueOf(adaptiveMedia.getURI()));`

Reply:
I actually previously had it as URI uri = adaptiveMedia.getURI() and long fileEntryId = _getFileEntryId(uri.toString()) but it tripped source formatter warnings. I'll try with the second suggestion.